homeassistant/components/zwave.py

```diff
@@ -183,9 +193,18 @@ def setup(hass, config):
     use_debug = str(config[DOMAIN].get(CONF_DEBUG)) == '1'
     customize = config[DOMAIN].get(CONF_CUSTOMIZE, {})
 
+    # Verify zwave device is writable
+    usb_stick_path = config[DOMAIN].get(CONF_USB_STICK_PATH,
+                                        DEFAULT_CONF_USB_STICK_PATH)
+    if not os.access(usb_stick_path, os.W_OK):
+        import getpass
+        _LOGGER.error("Current user '%s' does not have write permissions to"
+                      "usb device '%s'", getpass.getuser(), usb_stick_path)
+        return
+
```

Comment:
Turns out libopenzwave already does this. I'll leave it to them, unless we want to try for a more verbose error than what they return.